.github/pull_request_template.md

```diff
@@ -28,12 +28,15 @@ Pick one of the following and delete the others:
 - Misc (other changes)
 
 ###  How is the PR tested?
-
-Please describe the tests that you ran to verify your changes. Have you added unit or integration tests, or run manual tests? What additional tests would give you greater confidence in this change?
+Please select the tests that you ran to verify your changes:
+- Unit test
+- Integration test
+- Manual test (add detailed scripts or steps below)
+- No code
```

Comment:
What does "no code" mean?

Reply:
No code means did not have any code. then did not need to test

Reply:
the same with tidb